src/localization/en.json

```diff
@@ -29,7 +29,8 @@
     "dailyEntriesNotFound": "Could not find daily entries",
     "fetchFailedGeneral": "There was an error fetching data",
     "fetchFailedNoEntriesGeneral": "No time entries found",
-    "fetchSlackAvatarsFailed": "Error while fetching Slack avatars"
+    "fetchSlackAvatarsFailed": "Error while fetching Slack avatars",
+    "questionnaireSaveFailed": "Saving questionnaire has failed"
```

Comment:
Sticking with the usual structure of an error message I'd suggest:
"Error while saving questionnaire"